### PR TITLE
css to fix space issue

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/service-support/lab-automation-solutions/lab-automation-for-high-content-screening
- After: https://space-fix--moleculardevices--hlxsites.hlx.page/service-support/lab-automation-solutions/lab-automation-for-high-content-screening

### CODE REVIEW

**Review by mhaack — APPROVED** [review]

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -412,6 +416,18 @@ main .section>div.columns-wrapper {
...
+  margin-bottom: 0;
+}
+
+.layout-25-75.collaborators-column p:last-child {
+  margin-top: 0;
+}
+
+.layout-25-75.collaborators-column > div:not(:last-child) {
+  margin-bottom: 30px;
+}
```
Comment (mhaack) [inline blocks/columns/columns.css:430]: Does this only apply to the `collaborators-column` variant? Maybe the move it to `.collaborators-column ... {`

Comment (duynguyen) [reply blocks/columns/columns.css:430]: +1 you could also make `.collaborators-column .` styles for generic name and decoupled from layout-25-75 to allow reuse later.

**Review by duynguyen — APPROVED** [review]
LGTM. assuming that you already did some sanity checks on other pages @dev-rajneeshkumar @MeghnaTewari

**Comment by dev-rajneeshkumar** [comment]
Yes @duynguyen  thanks!

**Pull request merged**